Client/Coordinators/SettingsCoordinator.swift

```diff
@@ -128,7 +128,9 @@ class SettingsCoordinator: BaseCoordinator,
             return SearchBarSettingsViewController(viewModel: viewModel)
 
         case .topSites:
-            return TopSitesSettingsViewController()
+            let viewController = TopSitesSettingsViewController()
+            viewController.profile = profile
```

Comment:
I would like to make my feelings about this (because of how the settings are built, so not related to this work at all) known: I don't like it and one day it will change! haaaaaaaaaa laughcry

Reply:
Yes 😆 I agree, it was something I missed since it's not part of the `init`. One day!!